core/tracepoint/defined_class_spec.rb

```diff
@@ -0,0 +1,48 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.0' do
+  describe 'TracePoint#defined_class' do
+    module A; def bar; end; end
+
+    class B
+      include A
+      def foo; end;
+    end
+
+    class C < B
+      def initialize
```

Comment:
`initialize` is not useful here.

Reply:
I found a different behavior in specs on not having `initialize` --
last_class_name - `EqualMatcher` 
On having `initialize` - last_class_name  - `C` when --
C.new   
last_class_name.should equal(C)
Hence should we include `initialize` OR exclude checking last_class_name on `C.new`?

Reply:
Oh, my bad, it's fine to keep it then 😸